code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -125,24 +125,34 @@
 
 
 	var/appears_dead = FALSE
+	var/just_sleeping = FALSE
+
 	if(stat == DEAD || HAS_TRAIT(src, TRAIT_FAKEDEATH))
 		appears_dead = TRUE
+
+		var/obj/item/clothing/glasses/E = get_item_by_slot(slot_glasses)
+		var/are_we_in_weekend_at_bernies = E?.tint && buckled && istype(buckled, /obj/structure/chair/wheelchair)
```

Comment:
since we're using a fun variable name here, could you comment/document what exactly this variable does so it's clear to people who are reading this code down the line

Reply:
Done and done!